packs/circle_ci/actions/run_build.yaml

```diff
@@ -0,0 +1,15 @@
+name: run_build
+pack: circle_ci
+runner_type: run-python
+description: Run build for project and branch.
+enabled: true
+entry_point: run_build.py
+parameters:
+  project:
+    type: string
+    description: Name of project in circle ci.
+    required: true
+  branch:
```

Comment:
Their API is really awkward but atleast there is an API doc. I am still looking at the details to see what is possible.

There is support for SHA and tag but the URL is different.